lib/active_admin/filters/forms.rb

```diff
@@ -65,7 +65,7 @@ def active_admin_filters_form_for(search, filters, options = {})
           buttons = content_tag :div, class: "buttons" do
             f.submit(I18n.t('active_admin.filters.buttons.filter')) +
               link_to(I18n.t('active_admin.filters.buttons.clear'), '#', class: 'clear_filters_btn') +
-              hidden_field_tags_for(params, except: [:q, :page])
+              hidden_field_tags_for(params, except: [:q, :page, :current_user])
```

Comment:
so... its ready to be merged :stuck_out_tongue_winking_eye: 

if you see any place to add tests for it (I haven't), please tell so I can do it, otherwise we can go for it ;)